### PR TITLE
Carlos/be

### PR DESCRIPTION
**Pull request by Heart8reak**
completed Seed file & Migrations for Favorites Model

### CODE REVIEW

File: api/favorites/favoritesRouter.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const router = require('express').Router();
+
+const Favorites = require('./favoritesModel');
+// const City = require('../city/cityModel')
+// const Profile = require('../profile/profileModel')
+
```
Comment (cdlafortune) [inline api/favorites/favoritesRouter.js:4]: let's remove these variables if we don't need them right now

File: data/seeds/002_cities.js [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+          crime: 'medium',
+          air_quality_index: 'low',
+          diversity_index: '58',
+          walkability: '2.0',
+          livability: '85',
+          profile_id: '00ulthapbErVUwVJy4x6',
+        },
+      ]);
+    });
+};
```
Comment (cdlafortune) [inline data/seeds/002_cities.js:35]: glad we got this working!

**Review by cdlafortune — CHANGES_REQUESTED** [review]
Carlos, great job getting the seeds working! 
Per the Labs engineering standards, make sure you add a detailed description to your PRs.
Also, in the future, let's make branch names describe the feature you're working on - in this case, it could have been something like "knex-migrations".

File: api/favorites/favoritesRouter.js (hunk 1 above)
Comment (Heart8reak) [reply api/favorites/favoritesRouter.js:4]: ok done

**Review by cdlafortune — APPROVED** [review]
looks good!

**Review by Koyamot — APPROVED** [review]
Great work!

**Pull request merged**